tests/manage/rgw/conftest.py

```diff
@@ -16,7 +17,8 @@ def pytest_collection_modifyitems(items):
     """
     if (
         config.ENV_DATA['platform'].lower() not in ON_PREM_PLATFORMS
-        or float(config.ENV_DATA['ocs_version']) < 4.5
+        or skipif_ocs_version('<4.5')
+        or skipif_upgraded_from('4.4')
```

Comment:
maybe would be good to add a skip reason. If agreed, it will probably require changes in skipif_upgraded_from to accept a reason

Reply:
Also it would be good if `skipif_upgraded_from` accepted expressions like `skipif_ocs_version`. This PR currently covers only upgrade from OCS 4.4 but if upgrade to OCS 4.5 is triggered from older version (<4.4) then it's not skipped. For now we can use probably `['4.2', '4.3', '4.4']` as argument for `skipif_upgraded_from`.

Reply:
@ebenahar - I think it'd be better to just independently log the reason for the skip. I'll add it.
@fbalak - Great point, thank you, I'll add this too